.github/workflows/test-artifact-actions-and-stuff.yml

```
@@ -0,0 +1,24 @@
+name: Test artifact actions
+on: push
```

```suggestion
on: push
```

It's super great that you have added a workflow to test your action, but we should limit it to run only when the [action is modified](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-only-when-a-push-affects-specific-files)

---

Oh, I actually meant to remove that workflow before merging, should I keep it?

---

I don't mind keeping this, as this tests some actually vital functionality. @katexochen what's your opinion?

---

I'd opt to remove, code that doesn't run is eventually broken. The code will be tested well enough, as we execute it frequently in many actions.